action.yml

```diff
@@ -80,12 +80,20 @@ inputs:
       Please refer to this action's README for all available placeholders.
     default: >-
       [Backport ${target_branch}] ${pull_title}
+  pull_branchname:
+    description: ""
+    default: >-
+      backport-${pull_number}-to-${target}
   target_branches:
     description: >
       The action will backport the pull request to each specified target branch (space-delimited).
       Note that the pull request's headref is excluded automatically.
       Can be used in addition to backport labels.
       By default, only backport labels are used to specify the target branches.
+  target_owner:
+    description: ""
+  target_repo:
+    description: ""
```

Comment:
🔧 As it may be difficult to consider all different use cases, I suggest we move the `target_owner` and `target_repo` inputs to the experimental input. This will help clarify to users that this functionality is available but should be tried out carefully before fully trusting it. Once it has been available for some time, we can promote it to a fully supported feature.

Reply:
Moved it to the experimental input 👍